docs/content/deployment/run-coordinator.mdx

```diff
@@ -85,6 +85,24 @@ def less_important_schedule(_):
 
 A priority tag on the pipeline definition can be overriden when launching in the Dagit playground.
 
+#### Applying concurrency limits per unique value
+
+In some cases, you may want to apply separate limits to many possible values of a tag. Instead of enumerating each case, you can set a limit for each unique value.
+
+```yaml literalinclude showLines caption=dagster.yaml file=/deploying/concurrency_limits/per-unique-value-dagster.yaml
```

Comment:
im not sure if we support `literalinclude` anymore. the new code snippet api readme is here: https://github.com/dagster-io/dagster/tree/master/docs#code-snippets-literal-includes